packages/backend-common/src/scm/git.ts

```diff
@@ -150,12 +150,19 @@ export class Git {
     });
   }
 
-  async init({ dir }: { dir: string }): Promise<void> {
+  async init({
+    dir,
+    defaultBranch,
```

Comment:
I wonder if we should set a `defaultBranch = something` here too in order to not break this public API?

Reply:
Make sense! do you also want me to add the defaultBranch property in other providers as well?

Reply:
That makes sense, and yeah separate pr sounds good